bundler/spec/commands/newgem_spec.rb

```diff
@@ -1355,6 +1359,8 @@ def create_temporary_dir(dir)
 
     let(:require_relative_path) { "gem" }
 
+    let(:minitest_test_file_path) { "test/test/test_gem.rb" }
```

Comment:
Calling test gem "test-gem" was really bad idea :) It is really confusing here. Anyway we can change this later. It is not problem of this PR.